pyo3-ffi/src/cpython/unicodeobject.rs

```diff
@@ -137,11 +137,7 @@ pub unsafe fn PyUnicode_IS_COMPACT(op: *mut PyObject) -> c_uint {
 #[inline]
 #[cfg(target_endian = "little")]
 pub unsafe fn PyUnicode_IS_COMPACT_ASCII(op: *mut PyObject) -> c_uint {
-    if (*(op as *mut PyASCIIObject)).ascii() != 0 && PyUnicode_IS_COMPACT(op) != 0 {
-        1
-    } else {
-        0
-    }
+    ((*(op as *mut PyASCIIObject)).ascii() != 0 && PyUnicode_IS_COMPACT(op) != 0) as c_uint
```

Comment:
Wouldn't `.into()` work here as well? I think `Into` and `From` are generally preferable to casts where applicable?

Reply:
Sure, changed to into 👍